packages/nimble-components/karma.conf.js

```diff
@@ -29,7 +29,7 @@ const commonChromeFlags = [
     '--disable-infobars',
     '--disable-translate',
     '--force-prefers-reduced-motion',
-    '--lang=en'
+    '--lang=en-US'
```

Comment:
Just to sanity check, does this run correctly locally on a windows build? (not confirmed via CI which is linux)

Reply:
Yes, tests still pass when run locally on Windows.